packages/esm-patient-registration-app/package.json

```diff
@@ -8,7 +8,7 @@
   "license": "MPL-2.0",
   "homepage": "https://github.com/openmrs/openmrs-esm-patient-management#readme",
   "scripts": {
-    "start": "openmrs develop",
+    "start": "openmrs develop --backend https://ohri-demo.globalhealthapp.net",
```

Comment:
```suggestion
    "start": "openmrs develop",
```

Reply:
@brandones lol, why I'm this careless? Nice catch gonna remove that.

Reply:
@brandones resolved.